android/build.gradle

```diff
@@ -31,7 +31,7 @@ android {
         main.java.srcDirs += 'src/main/kotlin'
     }
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 21
```

Comment:
That’s a big jump, reason?

Reply:
android 21 is standard nowadays. Nobody supports back

Reply:
Vid app also supports from 21